contracts/Bond.sol

```diff
@@ -224,20 +224,18 @@ contract Bond is
             revert ZeroAmount();
         }
 
+        // saves an extra SLOAD
```

Comment:
can these be moved to below burn? they'd be closer to where they're used and in the case of burn reverting, there are gas savings 😉

Reply:
good idea - thanks for pointing it out.